### PR TITLE
Add Tests and GitHub Actions

### PR DESCRIPTION
**Pull request by martialblog**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: gomod
```
Comment (RincewindsHat) [inline .github/dependabot.yml]: Copy paste mistake. this is not golang software :-)

**Review by RincewindsHat — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (martialblog) [reply .github/dependabot.yml]: :hankey: :facepalm:

Comment (martialblog) [reply .github/dependabot.yml]: Fixed

File: README.md [hunk 2]
```diff
@@ -24,20 +29,26 @@ Then copy the file to your plugin dir:
...
+-h, --help            show this help message and exit
+--api API, -A API     HP MSA host url (e.g. https://msa1.local)
+--username USERNAME, -u USERNAME
+                      Username for login
+--password PASSWORD, -p PASSWORD
+                      Password for login
+--mode MODE, -m MODE  Check mode
+--insecure            Do not check certificates
+--version, -V         Print version
+--authentication_hash_algo {md5,sha256}
```
Comment (RincewindsHat) [inline README.md]:
```suggestion
  --auth_hash_type {md5,sha256}
```

File: README.md [hunk 3]
```diff
@@ -24,20 +29,26 @@ Then copy the file to your plugin dir:
...
+-h, --help            show this help message and exit
+--api API, -A API     HP MSA host url (e.g. https://msa1.local)
+--username USERNAME, -u USERNAME
+                      Username for login
+--password PASSWORD, -p PASSWORD
+                      Password for login
+--mode MODE, -m MODE  Check mode
+--insecure            Do not check certificates
+--version, -V         Print version
+  --auth_hash_type {md5,sha256}
```
Comment (RincewindsHat) [inline README.md]: I think I broke the indendation here.

**Review by RincewindsHat — CHANGES_REQUESTED** [review]
Looks good apart from that one minor thing (see comment)

File: README.md (hunk 3 above)
Comment (martialblog) [reply README.md]: Should already be fixed

**Review by RincewindsHat — APPROVED** [review]

**Pull request merged**